lib/fs.js

```diff
@@ -134,6 +134,10 @@ function maybeCallback(cb) {
   throw new ERR_INVALID_CALLBACK();
 }
 
+function isBuffersArray(value) {
+  return Array.isArray(value) && value.every((i) => Buffer.isBuffer(i));
```

Comment:
This should be a normal for-loop for better performance.

Reply:
Additionally, `isArrayBufferView()` should be used instead of `Buffer.isBuffer()` to allow other types.